docs/develop/json-manifest-overview.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Teams manifest for Office Add-ins (preview)
+description: Get an overview of the preview JSON manifest.
+ms.date: 05/24/2022
+ms.localizationpriority: high
+---
+
+# Teams manifest for Office Add-ins (preview)
+
+Microsoft is making a number of improvements to the Microsoft 365 developer platform. These improvements provide more consistency in the development, deployment, installation, and administration of all types of extensions of Microsoft 365, including Office Add-ins. These changes are compatible with existing add-ins. 
+
+Two important features we are aiming at, *but are not yet available*:
```

Comment:
Can we update the to focus on the shared manifest/schema to enable a single unit of distribution - it's more aligned to the scope of the preview.  We can add the other point as needed at a subsequent milestone. 

For example:
One important improvement we are working on is the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same shared manifest format and schema, based on the current JSON formatted Teams manifest schema.  

We have taken a first step towards....